bolt-contracts/config/holesky/deployments.json

```diff
@@ -19,7 +19,7 @@
         "avsDirectory": "0x055733000064333CaDDbC92763c58BF0192fFeBf",
         "delegationManager": "0xA44151489861Fe9e3055d95adC98FbD462B948e7",
         "strategyManager": "0xdfB5f6CE42aAA7830E94ECFCcAd411beF4d4D5b6",
-        "middleware": "",
+        "middleware": "0x279F5be63FD1D0A32d77258028933a7e3E75fF08",
```

Comment:
Let's keep this out of the config so we don't accidentally set incorrect addresses